test/Microsoft.AspNet.Mvc.Razor.Test/Precompilation/PrecompilationTagHelperTypeResolverTest.cs

```diff
@@ -0,0 +1,230 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.ComponentModel;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using Microsoft.AspNet.Mvc.Razor.Compilation;
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.Dnx.Compilation;
+using Microsoft.Dnx.Runtime.Infrastructure;
+using Microsoft.Framework.DependencyInjection;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.Razor.Precompilation
+{
+    public class PrecompilationTagHelperTypeResolverTest
+    {
+        private static readonly string TypeNamespace = typeof(PrecompilationTagHelperTypeResolverTest).Namespace;
+        private static readonly Assembly ExecutingAssembly = typeof(PrecompilationTagHelperTypeResolverTest).Assembly;
+        private static readonly TypeInfo TagHelperTypeInfo = typeof(ITagHelper).GetTypeInfo();
+
+        [Theory]
+        [InlineData("TypeDerivingFromITagHelper",
+            "Microsoft.AspNet.Mvc.Razor.Precompilation.TypeDerivingFromITagHelper")]
+        [InlineData("TypeInGlobalNamespace", "TypeInGlobalNamespace")]
```

Comment:
I am going to try and see if we can get it to both compile and be used as a resource.That way we can run same tests against `RuntimeTypeInfo` and `SymbolsBasedTypeInfo` to verify expectations match.